cli/cmd/init.go

```diff
@@ -43,7 +42,7 @@ func makeHappyClient(cmd *cobra.Command, sliceName, stackName, tag string, creat
 	if err != nil {
 		return nil, err
 	}
-	builderConfig := artifact_builder.
+	builderConfig := ab.
```

Comment:
Where is ab coming from?, names should be more descriptive than 1 letter, improves readability.

Reply:
It's a package name alias declared at the top of the file.